spec/controllers/courses_controller_spec.rb

```diff
@@ -308,4 +308,70 @@
       end
     end
   end
+
+  describe "#import_course" do
+    include_context "controllers shared context"
+    context "when user is administrator" do
+      before(:each) do
+        user = get_admin
+        sign_in(user)
+        @instructor_email = "instructor@gmail.com"
+        @course_name = "course"
+      end
+      it "successfully creates course from valid tar" do
```

Comment:
can you clean up the import course folder after the test?

Reply:
Once your PR is merged they will be removed bc the context is already included 👍